docs/alerting/api.md

```diff
@@ -858,3 +858,138 @@ _opendistro/_alerting/destinations/<destination-id>
   "_primary_term": 1
 }
 ```
+
+
+## Create email account
```

Comment:
Almost none of these API calls work. Many of your "responses" are actually sample request bodies. Use a tool like [Postman](https://www.postman.com/downloads/) to test the API, and copy and paste actual requests and responses.

Reply:
Will check them again and make sure they al work.

Reply:
I've corrected the request bodies. The sample responses are there for "create" email account and "create" email groups. For other sample responses you need to add an actual account ID. You can create one and replace <email_account_id> or use "RAau5XQBBF7ehDf33O3G" or "EQbQ5XQBBF7ehDf3CO7_" to replace <email_account_id>.